src/main/java/commands/ExitCommand.java

```diff
@@ -0,0 +1,23 @@
+package commands;
+
+import storage.Storage;
+import task.TaskList;
+/**
+ * Represents the command used to exit the application.
+ */
+public class ExitCommand extends Command {
+
+    public static final String COMMAND_WORD = "bye";
+    private static final String EXIT_MESSAGE = "Bye! Hope to see you again soon!";
+
+    /**
+     * Executes the ExitCommand, exiting the application
+     * @param tasks   The TaskList representing the collection of tasks.
+     * @param storage The Storage object handling storage operations.
+     * @return
+     */
```

Comment:
```suggestion
    /**
     * Executes the ExitCommand, exiting the application.
     * @param tasks   The TaskList representing the collection of tasks.
     * @param storage The Storage object handling storage operations.
     * @return Exit message.
     */
```
Perhaps adding a description for the return in the JavaDoc would be better?